.travis/is_spark_enrich_release_tag.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
As above, OT but is there a reason why we have all these individual is_${foo}_release_tag.sh scripts, when we could just have a singular is_release_tag.sh ${foo} approach?